subsys/bluetooth/common/Kconfig

```
@@ -4,6 +4,190 @@
 # Copyright (c) 2016 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+config BT_BUF_ACL_TX_SIZE
+	int "Maximum supported ACL size for outgoing data"
+	range 27 251
+	default 27
+	help
+	  Maximum supported ACL size of data packets sent from the Host to the
+	  controller. This value does not include the HCI ACL header.
+	  The Host will segment the data transmitted to the controller so that
+	  packets sent to the controller will contain data up to this size.
+	  In a combined build this value will be set in both the Host and the
+	  Controller.
+	  In a Host-only build the Host will read the maximum ACL size supported
+	  by the controller and use the smallest value supported by both the
+	  host and the controller.
+	  The Host supports sending of larger L2CAP PDUs than the ACL size and
+	  will fragment L2CAP PDUs into ACL data packets.
+	  The controller will return this value in the HCI LE Read Buffer
+	  Size command response. If this size if greater than effective Link
+	  Layer transmission size then the controller will perform
+	  fragmentation before transmitting the packet(s) on air.
+	  If this value is less than the effective Link Layer transmission size
+	  then this will restrict the maximum Link Layer transmission size.
+	  Maximum is set to 251 due to implementation limitations (use of
+	  uint8_t for length field in PDU buffer structure
+
+config BT_BUF_ACL_TX_COUNT
+	int "Number of outgoing ACL data buffers"
+	default 7 if BT_HCI_RAW
+	default 3
+	range 1 255
+	help
+	  Number of outgoing ACL data buffers sent from the Host to the.
+	  Controller. This determines the maximum amount of data packets the
+	  Host can have queued in the Controller before waiting for the
+	  to notify the Host that more packets can be queued with the Number of
+	  Completed Packets event.
+	  The buffers are shared between all of the connections and the Host
+	  determines how to divide the buffers between the connections.
+	  The controller will return this value in the HCI LE Read Buffer Size
+	  command response.
+	  In a 7.5ms connection interval and 2M PHY, maximum 19
+	  packets can be enqueued, with 18 packets with L2CAP payload size of 1
+	  byte can be acknowledged.
+
+config BT_BUF_ACL_RX_SIZE
+	int "Maximum supported ACL size for incoming data"
+	default 200 if BT_BREDR
+	# Mesh Proxy Recommended: 64 Pkey + 2 Bytes Mesh header.
+	# Overhead: Write command Header (3) in an L2CAP PDU (4).
```

```suggestion
	# Overhead: ATT Write command Header (3) in an L2CAP PDU (4).
```
Should we specify ATT here?